docs/adrs/004-SplitAndAggregate.md

```diff
@@ -125,12 +125,21 @@ Still to be determined (not in scope of this ADR):
 * `+` Simplest solution, quickest turnaround, fixes deadlock issue
 * `-` Addresses only the deadlocking issue, UI still broken for regular runs and batch runs 
 
-### Increase number of slots or Seperate Queue for kbparallels apps without 10 job limit
-* `+` Simple solutions, quick turnarounds, fixes deadlock issue
+### Increase number of slots per user > 10
+* `+` Simple solutions, quick turnarounds, fixes deadlock issue for small numbers of jobs.
+* `-` Doesn't fix deadlock issue as the user can still submit more KBP jobs
 * `-` Addresses only the deadlocking issue, UI still broken for regular runs and batch runs
 * `-` A small amount of users can take over the entire system
 * `-` The calculations done by the apps will interfere with other apps and cause crashes/failures
 
+###  Seperate Queue for kbparallels apps that may or may not have its own limit to running jobs.
+* `+` Simple solutions, quick turnarounds, fixes deadlock issue 
+* `+` Requires minimum changes to ee2 and condor if condor supports this feature
+* `-` Addresses only the deadlocking issue, UI still broken for regular runs and batch runs
+* `-` A small amount of users can take over the entire system unless  the new queue has its own limit to running jobs then it prevents users from taking over.
+* `-` The calculations done by the apps will interfere with other apps and cause crashes/failures
```

Comment:
This isn't true if it has its own queue, is it? Maybe I don't quite understand the separation between the queues.

Reply:
fixed

Reply:
changed

Reply:
From the new text:

> The calculations done by the apps will interfere with other apps in the new queue still and cause crashes/failures since they are doing more than managing the jobs right now. This would be solved by not allowing multple instances of apps that use KBP, but then it would waste a lot of resources.

I guess I don't understand how this is different from any other queue.

Reply:
I guess I didn't mention that if we had a new queue, we would need to take the nodes from somewhere, and if we want to make better use of the nodes we take, we'd want to have more jobs running per KBP node.  Changed it around